frontend/src/metabase/visualizations/visualizations/PivotTable/PivotTable.tsx

```diff
@@ -438,7 +443,10 @@ function PivotTable({
                         )
                       }
                       width={leftHeaderWidth}
-                      height={height - scrollBarOffsetSize()}
+                      height={
+                        (isDashboard ? height : queryBuilderGridHeight) -
```

Comment:
Seems like we need to wrap PivotTable component with ExplicitSize to have always correct `height` as we do with the CartesianChart

Reply:
I've already tried that, it does not work well because the `height` prop updates very slowly when narrowing a window, so the scrollbar ends up offscreen for a long time.